### PR TITLE
feat: add models for gitee.ai

### LINKED ISSUE #9243: please add model provider: gitee ai
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

please add model provider: [gitee ai](https://ai.gitee.com/models)

### 2. Additional context or comments

[Gitee AI ](https://ai.gitee.com/)汇聚了最新最热的 AI 模型，提供模型体验、推理、微调、部署和应用的一站式服务，我们提供了丰富的算力选择，希望帮助企业和开发者更容易地开发 AI 应用。

Gitee AI 是基于 Git 代码托管技术开发的针对 AI 应用场景的开源平台，基于 Gitee 多年在 Git 代码和 LFS 大文件托管方面的技术积累，我们为用户提供了快速、稳定的模型托管服务，让您轻松的分享和下载开源模型、数据集和 AI 应用，在此基础上我们目前构建了三个核心功能：Serverless API、模型引擎、应用，为您提供一站式的 AI 应用开发服务。

### 3. Can you help us with this feature?

- [X] I am interested in contributing to this feature.

### PR DESCRIPTION
**Pull request by fchange**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [X] Please open an issue before creating a PR or link to an existing issue
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

This pull request introduces support for Gitee AI as a new model provider. 

Close https://github.com/langgenius/dify/issues/9243
## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions
The changes were tested using manually filled credentials.
img_v3_02f3_41d4fba0-ccad-4750-80ee-9f55ddec38bg

### CODE REVIEW

**Review by crazywoola — CHANGES_REQUESTED** [review]
Tests are required for new providers.

File: api/core/model_runtime/model_providers/gitee_ai/gitee_ai.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import logging
+
+from core.model_runtime.model_providers.__base.model_provider import ModelProvider
+
+logger = logging.getLogger(__name__)
+
+
+class GiteeAIProvider(ModelProvider):
+    def validate_provider_credentials(self, credentials: dict) -> None:
+        pass
```
Comment (bluishoul) [inline api/core/model_runtime/model_providers/gitee_ai/gitee_ai.py]: Validate credential with API： https://ai.gitee.com/docs/openapi/v1#tag/account/GET/{namespace}/profile

**Comment by fchange** [comment]
Tests have been completed and verified.

**Comment by crazywoola** [comment]
> validate_credentials has been implemented and self-tested.

I mean you need to add tests here like this one 
https://github.com/langgenius/dify/blob/7c0b159a81f099c5089c96ba77b32d592e5787b8/api/tests/integration_tests/model_runtime/openai/test_llm.py

**Comment by fchange** [comment]
Added tests，because they use the real Gitee AI service, which requires a correctly filled access_token to run successfully. Therefore, the test command was not added to pytest_model_runtime.sh
@crazywoola

**Comment by bluishoul** [comment]
@crazywoola you can sign up Gitee AI, and use the free token(100 requests limit per day) to run the tests: https://ai.gitee.com/dashboard/settings/tokens

image

**Review by crazywoola — APPROVED** [review]

**Pull request merged**